package.json

```diff
@@ -24,7 +24,8 @@
     "eyeglass-module"
   ],
   "eyeglass": {
-    "exports": "lib/eyeglass-exports"
+    "exports": "lib/eyeglass-exports",
+    "needs": "*"
```

Comment:
Happy to add that after the first support lands. I totally agree that `*` is stupid, but as a plugin developer writing code explicitly for other people, maybe we expect better?